ui-admin/src/study/participants/participantList/ParticipantList.tsx

```diff
@@ -55,35 +57,60 @@ function ParticipantList({ studyEnvContext }: {studyEnvContext: StudyEnvContextT
 
   return <div className="ParticipantList container-fluid px-4 py-2">
     {renderPageHeader('Participant List')}
-    <div className="d-flex align-content-center align-items-center">
+    <div className="d-flex align-content-center align-items-center justify-content-between">
       <ParticipantSearch
         key={currentEnv.environmentName}
         studyEnvContext={studyEnvContext}
         searchState={searchState}
         updateSearchState={updateSearchState}
         setSearchState={setSearchState}
+        disabled={view === 'withdrawn'}
       />
-
-      {
-        familyLinkageEnabled && <div className="d-flex align-content-center p-2">
-          <Button
-            variant="light" className="border btn-sm"
-            aria-label={groupByFamily ? 'Participant view' : 'Family view'}
-            onClick={() => setGroupByFamily(groupByFamily ? 'false' : 'true')}>
-            {groupByFamily
-              ? <><FontAwesomeIcon size={'sm'} className={'p-0 m-0'} icon={faPerson}/> Participant view</>
-              : <><FontAwesomeIcon size={'sm'} className={'p-0 m-0'} icon={faPeopleGroup}/> Family view</>}
-          </Button>
-        </div>
-      }
-      <div><Link to={`withdrawn`}>Withdrawn</Link></div>
+      <div className="btn-group border my-1">
+        <Button id="grid" variant='light'
```

Comment:
why do we have two buttons with the same id of `grid`?  do we need the ids at all?

Reply:
ah this was copy/paste leftover from another button group, will clean up